lib/fastlane/plugin/revenuecat_internal/helper/github_helper.rb

```diff
@@ -8,7 +8,7 @@ module Fastlane
 
   module Helper
     class GitHubHelper
-      SUPPORTED_PR_LABELS = %w[breaking build ci docs feat fix perf refactor style test next_release dependencies minor].to_set
+      SUPPORTED_PR_LABELS = %w[breaking build ci docs feat fix perf refactor style test next_release dependencies phc_dependencies minor].to_set
```

Comment:
Nothing to do in this PR but we will need to add this label to the list of valid labels in danger